weave/conftest.py

```diff
@@ -56,14 +56,12 @@ def test_artifact_dir():
     return "/tmp/weave/pytest/%s" % os.environ.get("PYTEST_CURRENT_TEST")
 
 
-def pytest_sessionfinish(session, exitstatus):
```

Comment:
removing this busts all the integration tests because they skip clickhouse due to the vcr issue